apps/tlon-mobile/src/components/UrbitSigil.tsx

```diff
@@ -0,0 +1,39 @@
+import sigil from '@urbit/sigil-js/dist/core';
+import { useMemo } from 'react';
+import { View } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+import { useTailwind } from 'tailwind-rn';
+
+import { useIsDarkMode } from '../hooks/useIsDarkMode';
+
+type Props = {
+  ship: string;
+};
+
+export const UrbitSigil = ({ ship }: Props) => {
+  const tailwind = useTailwind();
+  const isDarkMode = useIsDarkMode();
+
+  const sigilXml = useMemo(
+    () =>
+      sigil({
+        point: ship,
+        detail: 'none',
+        size: 12,
+        space: 'none',
+        foreground: '#ffffff',
+        background: isDarkMode ? '#333333' : '#000000',
```

Comment:
Would be good to hook this up in a way that uses Tamagui themes/tokens system. `@tamagui/core` exports a  `useTheme` hook, which you can use to get current theme values, and there's also `@tamagui/get-token`, which lets you pull token values for sizing/spaces. You can also wrap this component in a Tamagui component that will resolve input tokens (see the `Icon` component).